backend/src/core/models/games.aggregate.js

```diff
@@ -1,46 +1,47 @@
 import { Game } from "./game.js";
 
 export class GamesAggregate {
-  games;
+  #games;
 
-  static manyFromDbEntries(games) {
-    const gamesAggregate = new GamesAggregate();
-    gamesAggregate.games = games.map((game) => Game.fromDbEntry(game));
+  constructor(games) {
+    this.#games = games.map((game) => Game.fromDbEntry(game));
+  }
 
-    return gamesAggregate;
+  get content() {
+    return this.#games;
```

Comment:
Always return or copy, because JS uses "pass by reference" of objects (and arrays which is an object). If you don't use a copy the returned value can be modified and by that the GamesAggregate instance also and you did not intended that.